Source/WTF/wtf/text/StringCommon.h

```diff
@@ -58,7 +58,11 @@ inline std::span<const char> span(const char* string)
     return { string, string ? strlen(string) : 0 };
 }
 
+#if HAVE(MISSING_U8STRING)
+inline std::span<const char8_t> span(const std::string& string)
+#else
 inline std::span<const char8_t> span(const std::u8string& string)
+#endif
 {
     return { string.data(), string.length() };
```

Comment:
doesn't `string.data()` give you a `const char*`? If so, how can you construct a `std::span<const char8_t>` from it? Is `char8_t` the same thing as `char` on Playstation?

Reply:
Whoops, thanks for noticing that. Was trying to simplify but went too far—I'll just use `span8()` instead.